Chart.yaml

```diff
@@ -1,13 +1,13 @@
 # apiVersion v2 is Helm 3
 apiVersion: v2
 name: airflow
-version: 1.8.7
+version: 1.9.0
 description: Helm chart to deploy the Astronomer Platform Airflow module
 icon: https://airflow.apache.org/docs/apache-airflow/stable/_images/pin_large.png
 keywords:
   - astronomer
   - airflow
 dependencies:
   - name: airflow
-    version: 1.8.0
-    repository: https://airflow.apache.org
+    version: 1.11.0-astro
+    repository: https://github.com/astronomer/airflow/releases/download/oss-helm-chart/1.11.0-astro
```

Comment:
why are we marking it to internal repo

Reply:
@pgvishnuram OSS airflow has not released a chart that has the fix we need. i have released a version of the OSS airflow chart that has the fix. when the public chart catches up, we will switch back. this is something we have done in the past. https://github.com/astronomer/airflow-chart/commit/1e5e98243b5b2c45dd350c9919c3998383a86561